components/malloc_size_of/lib.rs

```diff
@@ -967,7 +968,7 @@ impl<T> MallocSizeOf for hyper_serde::Serde<T> where
 
 // Placeholder for unique case where internals of Sender cannot be measured.
 // malloc size of is 0 macro complains about type supplied!
-impl<T> MallocSizeOf for std::sync::mpsc::Sender<T> {
+impl<T> MallocSizeOf for crossbeam_channel::Sender<T> {
     fn size_of(&self, _ops: &mut MallocSizeOfOps) -> usize {
         0
```

Comment:
The amount of memory allocated by the channel is at least:

```rust
match self.capacity() {
    None => self.len() * mem::size_of::<(bool, T)>(),
    Some(cap) => cap * mem::size_of::<(usize, T)>(),
}
```

By the way, since Servo may create multiple `Sender`s but only one `Receiver` per channel, shouldn't we be measuring the size of `Receiver`s rather than `Sender`s?